boost/teamcity_boost.cpp

```diff
@@ -71,7 +94,7 @@ struct TeamcityFormatterRegistrar {
     }
 };
 
-BOOST_GLOBAL_FIXTURE(TeamcityFormatterRegistrar)
+BOOST_GLOBAL_FIXTURE(TeamcityFormatterRegistrar);
 
 // Formatter implementation
 static std::string toString(boost::unit_test::const_string bstr) {
```

Comment:
good to you ;-) I did a lot of changes and now it is not possible to merge them in that way -- it is too late...